README.md

```diff
@@ -33,6 +33,7 @@
   <a href="https://gatsbyjs.org/docs/how-to-contribute">
     <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" alt="PRs welcome!" />
   </a>
+  <a href="https://twitter.com/intent/follow?screen_name=gatsbyjs"><img src="https://img.shields.io/twitter/follow/fbjest.svg?style=social&label=Follow%20@gatsbyjs" alt="Follow on Twitter"></a>
```

Comment:
```suggestion
  <a href="https://twitter.com/intent/follow?screen_name=gatsbyjs">
    <img src="https://img.shields.io/twitter/follow/gatsbyjs.svg?label=Follow @gatsbyjs" alt="Follow @gatsbyjs" />
  </a>
```

IMO the default, flat style looks better and matches the other badges.

Reply:
Done 👍